### ISSUE TITLE
Error upgrading from version 2.7.3 to 2.7.4

### ISSUE THREAD

**Issue by vindemasi** [centrifugal/centrifuge-js#139]
Hello,

after upgrading the library, this error happens:
```
Uncaught exception: SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at eval (Users/<...>/node_modules/centrifuge/dist/centrifuge.js:2445:21)
    at Array.map (<anonymous>)
    at JsonDecoder.decodeReplies (Users/<...>/node_modules/centrifuge/dist/centrifuge.js:2444:31)
    at Centrifuge._dataReceived (Users/<...>/node_modules/centrifuge/dist/centrifuge.js:885:35)
    at WebSocket._transport.onmessage (Users/<...>/node_modules/centrifuge/dist/centrifuge.js:705:16)
    at WebSocket.onMessage (Users/<...>/node_modules/ws/lib/event-target.js:132:16)
    at WebSocket.emit (events.js:315:20)
    at Receiver.receiverOnMessage (Users/<...>/node_modules/ws/lib/websocket.js:825:20)
    at Receiver.emit (events.js:315:20)
    at Receiver.dataMessage (Users/<...>/node_modules/ws/lib/receiver.js:437:14)
    at Receiver.getData (Users/<...>/node_modules/ws/lib/receiver.js:367:17)
    at Receiver.startLoop (Users/<...>/node_modules/ws/lib/receiver.js:143:22)
    at Receiver._write (Users/<...>/node_modules/ws/lib/receiver.js:78:10)
    at writeOrBuffer (internal/streams/writable.js:358:12)
    at Receiver.Writable.write (internal/streams/writable.js:303:10)
    at Socket.socketOnData (Users/<...>/node_modules/ws/lib/websocket.js:900:35)
    at Socket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
    at Socket.Readable.push (internal/streams/readable.js:223:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)
```

The application is running under electron 12.0.2 (it's powered by node 14). The same upgrade in an Angular web app application is working fine.

Have a nice day.

**Comment by vindemasi** [comment]
Well,

just put some console.log shown the issue:

```
DATA: {"id":1,"result":{"client":"21291bc6-c571-4eca-a49f-4395d15d2e6e","version":"2.8.4","expires":true,"ttl":86400}}

split: {"id":1,"result":{"client":"21291bc6-c571-4eca-a49f-4395d15d2e6e","version":"2.8.4","expires":true,"ttl":86400}}
split:
```

it's seems there some spurious new line. I think a simple trim should do the work.

**Comment by FZambia** [comment]
@vindemasi many thanks for opening an issue and investigating root cause.

First of all - so sorry for this, the reason why I missed this is because I tested that change with recent Centrifuge library where I removed a `\n` in the end of JSON. That's why it worked for me - but since Centrifugo did not inherit these changes yet - it crashed everywhere else.

I just released 2.7.5 with a fix - could you please try it?

**Comment by vindemasi** [comment]
@FZambia thank you for really quick fix. It works perfectly.
